airtouch4pyapi/communicate.py

```diff
@@ -56,20 +74,21 @@ def TranslateMapValueToValue(groupChunk, map):
     return byteSegmentAsValue
 
 #might raise a socket or os error if connection fails
-async def SendMessagePacketToAirtouch(messageString, ipAddress):
+async def SendMessagePacketToAirtouch(messageString, ipAddress, atVersion):
     #add header, add crc
-    messageString = "5555" + messageString + format(crc16(bytes.fromhex(messageString)), '08x')[4:]
```

Comment:
This can't change unfortunately - airtouch 4 needs this to just be 5555

Reply:
Fixed (I think)